apps/logging/signals.py

```diff
@@ -89,10 +92,14 @@ def log_grant_removed(sender, instance=None, **kwargs):
     token_logger.info(get_event(DataAccessGrantSerializer(instance, action="revoked")))
 
 
+@receiver(pre_fetch, sender=FhirDataView)
+@receiver(pre_fetch, sender=FhirServerAuth)
 def fetching_data(sender, request=None, **kwargs):
     fhir_logger.info(get_event(FHIRRequest(request)))
```

Comment:
@JFU-GIT Wondering if a "type" field could be added to the logging here to better distinguish the type?  

For example, this could include a type="fhir_pre_fetch" and for the post_fetch type="fhir_post_fetch".

The code could be something like this, and also updated in the serializer:

```python
    fhir_logger.info(get_event(FHIRRequest(request, "fhir_pre_fetch")))
```
The string value of the sender class info might be useful also.

Reply:
good point, that way audit events will manifest themselves

Reply:
now the logged event has type: fhir_pre_fetch / fhir_post_fetch